fix_DP_audio.sh

```diff
@@ -2,5 +2,5 @@
 
 if [ ! $(grep -qi -e "ZCU102" -e "ADRV9009-ZU" -e "Jupiter SDR" /sys/firmware/devicetree/base/model) ]; then
 	# modify audio default sampling rate
-	sed -i '/default-sample-rate/c\default-sample-rate = 48000' /etc/pulse/daemon.conf
+	sed -i '/default-sample-rate/c\default-sample-rate = 48000' /etc/pulse/daemon.conf 2>/dev/null || true
```

Comment:
Why do you need "|| true" at the end ?

Reply:
Because even if the error message is suppressed, the command still exits with an error code in case the file does not exist, making the script and service to fail. "|| true" is used to ignore the exit code of the "sed" command.